app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -28,6 +28,9 @@ class << model_name
     has_many :key_pairs,                     :class_name  => "AuthPrivateKey", :as => :resource, :dependent => :destroy
     has_many :host_aggregates,               :foreign_key => :ems_id, :dependent => :destroy
 
+    virtual_has_many :cloud_networks,  :uses => {:network_manager => :cloud_networks}
+    virtual_has_many :security_groups, :uses => {:network_manager => :security_groups}
```

Comment:
I'm not sure how cloud_networks is written, but can this leverage a real relation: `has_many :cloud_networks, :through => :network_manager` ?